github/repos_commits.go

```diff
@@ -146,6 +147,35 @@ func (s *RepositoriesService) GetCommit(owner, repo, sha string) (*RepositoryCom
 	return commit, resp, err
 }
 
+// GetCommitSHA1 gets the SHA-1 of a commit reference.
+// If a last-known SHA1 is supplied and no new commits have occurred, a
```

Comment:
If the intent is for this to be a separate paragraph, I think it would need a blank line before it.  If not, why have it separated from the line above?